core/chains/evm/forwarders/forwarder_manager_test.go

```diff
@@ -32,10 +33,12 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/forwarders"
 )
 
+var overrideDefaultID = func(c *toml.EVMConfig) { c.ChainID = ubig.New(testutils.FixtureChainID) }
```

Comment:
Can this value just be the default so that callers don't have to pass this func every time?